taxi/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Driver, Manufacturer
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer", ]

```

Comment:
Can you add list_display for more info about cat?

Reply:
![1_6](https://github.com/mate-academy/py-taxi-service-startproject/assets/137777904/369ea795-d35f-4966-8891-4ed64609ab79)

Reply:
Thx, it's a great point!)